features/subscription_attach_restrictions.feature

```diff
@@ -0,0 +1,55 @@
+@uses.config.contract_token
+Feature: One time pro subscription related tests
+
+  Scenario Outline: Attach fail if subscription is restricted to release
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I set the machine token overlay to the following yaml:
+      """
+      machineTokenInfo:
+          contractInfo:
+              resourceEntitlements:
+                  - type: support
+                    affordances:
+                        onlySeries: <onlyseries>
+      """
+    When I attach `contract_token` with sudo
+    Then stderr contains substring:
+      """
+      Attaching to this contract is only allowed on the Ubuntu <onlyseries> <onlyrelease> release
```

Comment:
In other scenarios related to services where we identify the release and series like this, we use a different format.
`<release> LTS (series_full_name)` (which is the VERSION string format in /etc/os-release)

Could we, instead of
`Ubuntu jammy 22.04`
say
`Ubuntu 22.04 LTS (Jammy Jellyfish)`
for consistency reasons?

Reply:
Thanks, this is good - the current string was taken from the spec so it's my bad for not following our convention.